### PR TITLE
Unit Tests to validate v2 Full endpoint

### PR DESCRIPTION
**Pull request by EmilyM1**

### CODE REVIEW

File: pkg/chargeback/http_test.go [hunk 1]
```diff
@@ -333,3 +335,282 @@ func TestAPIV1ReportsGet(t *testing.T) {
...
+		expectedStatusCode int
+		expectedAPIError   string
+
+		queryerPrepareFunc func(*mockpresto.MockExecQueryer) []presto.Row
+		expectedResults    *GetReportResults
+	}{
+		"report-finished-with-results-full": {
+			reportName: testReportName,
+			report:     newTestReport(testReportName, namespace, testQueryName, reportStart, reportEnd, v1alpha1.ReportStatus{Phase: v1alpha1.ReportPhaseFinished}),
+			query: newTestReportGenQuery(testQueryName, namespace, []v1alpha1.ReportGenerationQueryColumn{
```
Comment (chancez) [inline pkg/chargeback/http_test.go]: This query should have one field with tableHidden true, and one with false like the others to test that this is in fact returning columns with table hidden to true.

**Comment by chancez** [comment]
The one query needs updating, but looks good otherwise @EmilyM1

File: pkg/chargeback/http_test.go (hunk 1 above)
Comment (EmilyM1) [reply pkg/chargeback/http_test.go]: Great catch! Thank you

**Comment by chancez** [comment]
Failure was a flake on the GKE cluster, just going to re-trigger the build

**Pull request merged**